### DISCUSSION TITLE
Didi index

### DISCUSSION THREAD

**Discussion by lucasbaraoneves** [DaveSkender/Stock.Indicators#610] [Ideas]
Hi, is it possible to implement the Didi Index?

**Comment by DaveSkender** [comment]
I can take a look at it.  Do you have any reputable online references that describe it?  Please do not send community-developed TradingView or MetaTrader scripts, unless they are from the original author; I'm looking for the published recipes from the original author/creator.

**Reply by lucasbaraoneves** [reply]
I really tried to find the concept from the original author but no success. 
Didi Index was developed by the brazilian trader Didi Aguiar.

I found a lot of articles but almost all in portuguese. The only english article that can better explain the Didi Index came from Trading View.

If possible, I can provide the brazilian articles here and do a simple translation.

**Reply by DaveSkender** [reply]
I'll take a translation if it can be done accurately.  I do see the original author has a website and _appears_ to be a recognized pro.  As long as a reputable and recognized industry expert has intentionally published it for public consumption, we can consider it for the library.

I'll have to take a closer look though.  If it's just usage of existing indicators in a specific combo, it gets into that gray area of "strategy and analysis" where I have to think carefully about whether it's in-scope of the library.
